packages/lit-dev-content/site/tutorials/content/wc-to-lit/03.md

```diff
@@ -0,0 +1,92 @@
+
+For example, try putting a style tag in the head of the main document:
+
+### index.html
+
+```html
+<!DOCTYPE html>
+<html>
+ <head>
+   <script src="./index.js" type="module"></script>
+   <style>
+     span {
+       border: 1px solid red;
+     }
+   </style>
+ </head>
+ <body>
+   <rating-element></rating-element>
+ </body>
+</html>
+```
+
+Your output should have a red border box around the span for the rating. This is a trivial case, but the lack of DOM encapsulation may result in larger problems for more complex applications. This is where Shadow DOM comes in.
+
+## Attaching a Shadow Root
+
+Inside the `connectedCallback`, attach a Shadow Root to the element and render the DOM inside of that root:
+
+### index.js
+
+```js
+connectedCallback() {
+  const shadowRoot = this.attachShadow({mode: 'open'});
+
+  // Note the new CSS Selectors
+  shadowRoot.innerHTML = `
+    <style>
+      :host {
+        display: inline-flex;
+        align-items: center;
+      }
+      button {
+        background: transparent;
+        border: none;
+        cursor: pointer;
+      }
+    </style>
+    <button class="thumb_down" >
+      <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M15 3H6c-.83 0-1.54.5-1.84 1.22l-3.02 7.05c-.09.23-.14.47-.14.73v2c0 1.1.9 2 2 2h6.31l-.95 4.57-.03.32c0 .41.17.79.44 1.06L9.83 23l6.59-6.59c.36-.36.58-.86.58-1.41V5c0-1.1-.9-2-2-2zm4 0v12h4V3h-4z"/></svg>
+    </button>
+    <span class="rating">${this.rating}</span>
+    <button class="thumb_up">
+      <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M1 21h4V9H1v12zm22-11c0-1.1-.9-2-2-2h-6.31l.95-4.57.03-.32c0-.41-.17-.79-.44-1.06L14.17 1 7.59 7.59C7.22 7.95 7 8.45 7 9v10c0 1.1.9 2 2 2h9c.83 0 1.54-.5 1.84-1.22l3.02-7.05c.09-.23.14-.47.14-.73v-2z"/></svg>
+    </button>
+  `;
+}
+```
+
+When the page refreshes, you will notice that the styles in the main document can no longer select the nodes inside the Shadow Root.
+
+How did you do this? In the `connectedCallback` you called `this.attachShadow` which attaches a shadow root to an element. The `open` mode means that the shadow content is inspectable and makes the shadow root accessible via `this.shadowRoot` as well. Take a look at the Web Component in the Chrome inspector as well:
+
+<img src="/images/tutorials/wc-to-lit/dom-tree.png" alt="The dom tree in the chrome inspector. There is a <rating-element> with a#shadow-root (open) as it&rsquo;s child, and the DOM from before inside that shadowroot.">
+
+You should now see an expandable shadow root that holds the contents. Everything inside that shadow root is called the *Shadow DOM*. If you were to select the rating element in Chrome Dev Tools and call `$0.children`, you will notice that it returns no children. This is because Shadow DOM is not considered a part of the same DOM tree as direct children but rather the *Shadow Tree*.
```

Comment:
In this one tutorial page, we've used shadow root, shadow DOM, and shadow tree. I'm not sure it's worth introducing the term shadow tree here, esp. since we go back to shadow DOM afterwards. (Technically, shadow tree may be more correct, but we tend to use shadow DOM because it's more familiar.) 

Also, _expandable_ could be confusing here--not clear whether it refers to the shadow root itself or the DevTools representation of the shadow root. 


```suggestion
You should now see a shadow root inside your component. You can expand the shadow root to show the contents of the shadow DOM. If you select the rating element in Chrome Dev Tools and call `$0.children`, you'll notice that it returns no children. The shadow DOM is its own DOM tree, separate from the component's direct children.
```